plugin/init.lua

```diff
@@ -369,30 +391,102 @@ function pub.fuzzy_load(window, pane, callback, opts)
 		end
 	end
 
-	local function insert_choices(type, fmt)
-		for _, file in ipairs(wezterm.glob("*", pub.save_state_dir .. "/" .. type)) do
-			local label
+	local function file_datetime(folder, file)
+		-- Month names
+		local months = {
+			"Jan",
+			"Feb",
+			"Mar",
+			"Apr",
+			"May",
+			"Jun",
+			"Jul",
+			"Aug",
+			"Sep",
+			"Oct",
+			"Nov",
+			"Dec",
+		}
+		local result = ""
+		if is_windows then
+			local handle = io.popen("wmic datafile where name='" .. folder .. separator .. file("' get LastModified"))
```

Comment:
Does this work? The last part of the line: `file("' get LastModified"))` seems to do a function call to the file? This does not seem right

Reply:
To be honest I've worked with ChatGPT, but I tried the command on a virtual Windows machine and it worked. But as I said in the initial comment, I did not test extensively on windows, I just made sure that the commands I was generating would work in the shell (cmd not powershell to make it more portable I hope).

Reply:
No worries, i think it should be changed to:
```suggestion
			local handle = io.popen("wmic datafile where name='" .. folder .. separator .. file .. "' get LastModified")
```